### PR TITLE
feat: locales usage in docs

### PR DESCRIPTION
**Pull request by EvilG-MC**
lol

### CODE REVIEW

**Review by socram03 — DISMISSED** [review]

File: sidebar.items.mjs [hunk 1]
```diff
@@ -193,6 +209,13 @@ export default [
...
             },
             link: '/recipes/transformers'
+          },
+          {
+            label: 'Locales',
+            translations: {
+              es: 'Localización'
+            },
+            link: '/recipes/locales'
           }
```
Comment (socram03) [inline sidebar.items.mjs]:
```suggestion
```
I don't think this is necessary

Comment (EvilG-MC) [reply sidebar.items.mjs]: Yup, I saw that when I was checking, it shouldn't be there anymore

File: src/content/docs/components/polls.mdx [hunk 2]
```diff
@@ -69,9 +69,9 @@ Here is a quick example that edits the message when a poll is completed:
 ```ts title='messageUpdate.ts' showLineNumbers
 export default createEvent({
     data: { name: 'messageUpdate' },
-    //This is [oldMessage, newMessage]
-    //But in this example we only need newMessage
-    run: ([, newMessage]) => {
+    run: ([oldMessage, newMessage]) => {
+        if (oldMessage?.id === newMessage.id) return;
```
Comment (socram03) [inline src/content/docs/components/polls.mdx]: XD?

**Review by socram03 — DISMISSED** [review]

File: src/content/docs/components/polls.mdx (hunk 2 above)
Comment (EvilG-MC) [reply src/content/docs/components/polls.mdx]: Se me escapó 🗣️

**Review by FreeAoi — APPROVED** [review]

**Review by socram03 — APPROVED** [review]

**Pull request merged**